### ISSUE TITLE
Tests failing: Undefined array key "gc__taxonomy_term"

### ISSUE THREAD

**Issue by stephen-cox** [localgovdrupal/localgov_microsites_group#376]
Tests are failing with `Exception when installing config for module group, message was: Undefined array key "gc__taxonomy_term"`

Looks to be a result of merging #372

**Comment by finnlewis** [comment]
Aha - here's a reference to the `gc__taxonomy_term` error.

@ekes do you know of any differences that might have introduced this with https://github.com/localgovdrupal/localgov_microsites_group/pull/372 ?

**Comment by ekes** [comment]
> Aha - here's a reference to the `gc__taxonomy_term` error.
> 
> @ekes do you know of any differences that might have introduced this with #372 ?

#372 does:-

https://github.com/localgovdrupal/localgov_microsites_group/pull/372/files
```
-    "drupal/group_term": "dev-3316529-group-3.x-compatibility#149bf98c",
+    "drupal/group_term": "4.0.0-alpha2",
```
So checking out git@git.drupal.org:issue/group_term-3316529.git @ 149bf98c, and adding the remote for the main module, and then diff'ing. Ohh! There's lots of stuffs. But it's mainly exported views, and text changes.

The exported views.view.group_terms does mention (in both versions `gc__taxonomy_term`)
[group-term-3316529at149bf98.diff.txt](https://github.com/localgovdrupal/localgov_microsites_group/files/10447587/group-term-3316529at149bf98.diff.txt)

**Comment by ekes** [comment]
https://github.com/localgovdrupal/localgov_microsites_group/blob/2.x/composer.json#L14 should be alpha3 by now btw

**Comment by finnlewis** [comment]
I tested yesterday with the original branch from #372 (https://git.drupalcode.org/issue/group_term-3316529/-/tree/3316529-group-3.x-compatibility) and I got the same failures locally, so it does not seem to be directly due to the switch.

**Comment by finnlewis** [comment]
I've posted an issue over at https://www.drupal.org/project/group_term/issues/3334882 

I've tried removing everything from the test, and just adding the dependency on localgov_microsites_events is enough to trigger the error. 

Some bad config in there perhaps? Or a problem with the group_term module.

```
/**
   * {@inheritdoc}
   */
  protected static $modules = [
    'localgov_microsites_events',
  ];
```

**Comment by finnlewis** [comment]
@ekes ekes has confirmed that this error is not just in LocalGov Drupal tests, but it shows in phpunit tests on group_term module on Drupal.org https://www.drupal.org/project/group_term/issues/3334882#comment-14882986 

So if anyone comes here looking for a solution, go there and try to help find one! 

He's added some tests that exhibit this behaviour here: https://git.drupalcode.org/issue/group_term-3334882/-/blob/3334882-undefined-array-key/tests/src/Functional/EntityOperationsTest.php

**Comment by stephen-cox** [comment]
I'm hoping the issue is caused by the view that comes with the Group Term module. I have rebuild this and patched the module with #392.
